docs/performance.md

```diff
@@ -0,0 +1,53 @@
+# Performance
+
+## Rerenders
+Duplicate rerenders of React components is a common
+performance bottleneck in React Native apps.
+
+Try to follow these tips to avoid needless rerenders of your components.
+
+1. Divide your components into Presentational and Container components. This helps in separation of concerns and also increase performance.
```

Comment:
sorry, I couldn't find any open source projects which use this separation and naming scheme explicitly, but I added a link to redux examples here https://github.com/reactjs/redux/tree/master/examples which would clarify the idea even more.